components/hero.tsx

```diff
@@ -33,34 +36,63 @@ export default function Hero() {
         {/* Hero content */}
         <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
           {/* Section header */}
-          <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Pear: The Open-Source AI-Powered Code Editor
-            </h1>
-            <p
-              className="mb-8 text-xl text-secondary-main"
-              data-aos="fade-up"
-              data-aos-delay="200"
-            >
-              Supercharge your development in an editor designed for less
-              coding, with AI.
-            </p>
+          <div className="mx-auto flex max-w-3xl flex-col items-center pb-12 text-center md:pb-16">
+            <div className="flex flex-col space-x-2 text-3xl font-semibold md:items-center md:text-4xl">
+              <div className="flex flex-row items-center space-x-2">
+                <PearHeroLogo width="26" className="hidden sm:block" />
```

Comment:
It's a good practice to provide alternative text (alt attribute) for the <PearHeroLogo> component just in case the image fails to load or for screen reader users

Reply:
that would be for both PearHeroLogo components, and the same alt text can be used for both

Reply:
Dont!